scripts/dapp-gc-voting.Dockerfile

```diff
@@ -1,6 +1,6 @@
 # This dockerfile is meant to be run from the **root of the repository**.
 
-ARG build_image=node:18-slim
+ARG build_image=node:18.14-slim
```

Comment:
Why is the minor version necessary?

Reply:
I got an error from `tsx` (a node dev dependency for executing typescript), which I tracked down to being due to the node minor version